src/Simulation/Core/RuntimeMetadata.cs

```diff
@@ -37,10 +39,15 @@ public class RuntimeMetadata
         /// <summary>
         /// True if operation is composed of multiple operations.
         /// </summary>
+        /// </summary>
+        /// <remarks>
+        /// Currently not used as this is intended for compositeoperations,
```

Comment:
What does this even mean in the base class?

Reply:
Not sure I understand the question. Do you mean the default case? In that case this would be `false`.